store/index.js

```diff
@@ -92,7 +92,7 @@ export const mutations = {
 export const actions = {
   async height ({ rootGetters: { middleware }, commit }) {
     try {
-      const { mdwHeight: height } = await middleware.getStatus()
+      const { nodeHeight: height } = await middleware.getStatus()
```

Comment:
The result of the middleware status endpoint is like
```
{
  "mdw_height": 489861,
  "mdw_synced": true,
  "mdw_syncing": true,
  "mdw_tx_index": 26103628,
  "mdw_version": "1.0.9",
  "node_height": 489862,
  "node_progress": 100,
  "node_revision": "8a1ba3d845bb2c03dcdd06bd74fcd98610ffcdba",
  "node_syncing": false,
  "node_version": "6.2.0"
}
```

I think getting the node height is the right one and it should solve the problem. @marc0olo can you confirm?

Reply:
I will have a short check again. something was definitely wrong when I checked last time. my guess is we have a calculation error somehow.

I don't think it makes sense to use the node height for displaying that because the rest of the frontend is only aware of data that is returned by the middleware. so you should revert this IMO.